pkg/volume/util/operationexecutor/operation_generator.go

```diff
@@ -233,39 +234,61 @@ func (og *operationGenerator) GenerateAttachVolumeFunc(
 	}, nil
 }
 
+func (og *operationGenerator) splitUniqueName(uniqueName v1.UniqueVolumeName) (string, string, error) {
+	components := strings.SplitN(string(uniqueName), "/", 3)
```

Comment:
Please document expected format of uniqueName and link it to `GetUniqueVolumeNameFromSpec`.

Also, the code here expects that all volume plugin names contain exactly one '/'. This is OK with current code, however comment in plugin interface definition at https://github.com/kubernetes/kubernetes/blob/master/pkg/volume/plugins.go#L74 should be more strict - from 'should use namespaced names such as "example.com/volume"' to 'must use namespaced names ... and must contain exactly one "/" character.'